src/@carbon-info/views/IntroPage/IntroPage.tsx

```diff
@@ -158,7 +166,25 @@ const useStyles = makeStyles((theme: Theme) => ({
     fontWeight: 600,
     color: theme.palette.text.hint,
   },
+  altCarbonContainer: {
+    width: "100%",
+    display: "flex",
+    justifyContent: "flex-end",
+    cursor: "pointer",
+    marginTop: "-10rem",
+    "@media (max-width: 790px)": {
+      marginTop: "0rem",
+    },
+    zIndex: 5,
+  },
+  altCarbon: {
+    border: "1px solid #FFFFFF",
```

Comment:
The border color is `theme.palette.text.secondary`. Also, don't forget to add box-shadow `33px 33px 75px -10px rgba(0, 0, 0, 0.75), -33px -33px 75px -10px rgba(0, 0, 0, 0.33)`.
<img width="468" alt="Screenshot 2023-07-31 at 4 07 41 PM" src="https://github.com/Switcheo/carbon-website/assets/65583522/46debb53-48c4-4df0-afaa-fa5a287fd152">